### DISCUSSION TITLE
Settings for gmail

### DISCUSSION THREAD

**Discussion by CaliLuke** [moralmunky/Home-Assistant-Mail-And-Packages#641] [Q&A] [answered]
I might be dense but what is the correct "host" setting for run of the mill Gmail? I tried mail.google.com and gmail.com without success.

Thanks

**Comment by firstof9** [comment]
This is [covered in the wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Configuration-and-Email-Settings#gmail), the second line.

**Reply by CaliLuke** [reply]
Sorry but I still don't see it. This is the entirety of the content in the Gmail section:

```
GMail
Please note if you are using GMail for [this](https://support.google.com/accounts/answer/185833?hl=en) you will need to setup an App Password in order for this to work. Please refer to this link for details on how to setup an App Password

Additional information on GMail settings are [here](https://support.google.com/a/answer/9003945#zippy=%2Cstep-set-up-gmail-with-the-older-version-of-outlook) under "Set up Gmail with older versions of Outlook and other clients".
```

do you mean that the answer is buried in one of the links? I might have missed that, but I still think the docs could make key info visible in the main page with very little effort, happy to contribute an edit if that's possible.

**Comment by firstof9** [comment, marked as answer]
Additional information on GMail settings are [here](https://support.google.com/a/answer/9003945#zippy=%2Cstep-set-up-gmail-with-the-older-version-of-outlook) under "Set up Gmail with older versions of Outlook and other clients".
